### ISSUE TITLE
bug in version comparison for tikaServerTextExtractor

### ISSUE THREAD

**Issue by alwex** [silverstripe/silverstripe-textextraction#32]
Hi,

It seems that there is a bug with version comparison in the `TikaServerTextExtractor` class on the `isAvailable()` method. The version compatibility check fail silently if version is >= 1.1x.*

The code is as follow

```php
public function isAvailable()
{
    return $this->getServerEndpoint() &&
    $this->getClient()->isAvailable() &&
    $this->getVersion() >= 1.7;
}
```
I am using tika server v1.13.0 and 1.13 < 1.7

a suggestion would be to use version_compare php built in function to fix this issue.

```php
public function isAvailable()
{
    return $this->getServerEndpoint() &&
    $this->getClient()->isAvailable() &&
    version_compare($this->getVersion(), '1.7.0') >= 0;
}
```
see http://php.net/manual/fr/function.version-compare.php

**Comment by tractorcow** [comment]
Yes that's right; Version comparison should not be using numeric comparison only. :)

Would you mind please raising a PR and I'll merge.

**Comment by tractorcow** [comment]
Fixed with #33
